client/my-sites/checkout/checkout-thank-you/header.jsx

```diff
@@ -138,64 +87,13 @@ export class CheckoutThankYouHeader extends PureComponent {
 			);
 		}
 
```

Comment:
Removing the basic domain transfer code, but leaving the nested condition for delayed transfers in place, as I believe those don't pass flow through the refactored congrats pages.